rbd/rbd_nautilus.go

```diff
@@ -45,3 +47,21 @@ func GetImageNames(ioctx *rados.IOContext) ([]string, error) {
 	}
 	return names, nil
 }
+
+// GetCreateTimestamp returns the time the rbd image was created.
+//
+// Implements:
+//  int rbd_get_create_timestamp(rbd_image_t image, struct timespec *timestamp);
+func (image *Image) GetCreateTimestamp() (timespec Timespec, err error) {
```

Comment:
If you don't use the return declaration, I'd prefer to not do it. It becomes confusing later if you have to refactor if you do both in one function. Either do bare returns and use theses vars or drop them please [1].

[1] - I realize I haven't been complaining about this as much in the past but recent work on other parts of the code made me notice it more recently.

Reply:
Ah, yeah, introduced by copy/pasting some other function...